### ISSUE TITLE
Some people can't enter their own lands, Please Help

### ISSUE THREAD

**Issue by FAKUgaming** [GriefPrevention/GriefPrevention#610]
I have a handful of people on my server that are having issues entering their houses through doors even in claimed lands. It prevents them from entering their own claims, this is random as it is only effecting certain people and not all as well as only having this issue with doors and once the door is placed. If no door was placed, there is no issue.
A temp fix this that I have found out is that if they destroy the door and the blocks it was connected to and put the blocks back down and replace the door. This fix only works until the server restarts though and which they are locked back out. 
I have yet to be able to replicate this on my account even if I'm not OP'd, but I have witnessed seeing a player having issues getting into their own home. 

Can anyone explain this to me on why it keeps doing this to only a select few people? 
Is there a way that I can disable this feature?

### GriefPrevention config.yml
`https://gist.github.com/FAKUgaming/0b6d8fe854073b9e72c65e0a002aa5ad`

### Plugin list (if applicable):
AntiXRay
LaggRemover
NoCheatPlus
PlugMan
PluginMetrics
ServerRestartWarning

I am sorry If I missed any info or just oof and clearly missed something.

**Comment by RoboMWM** [comment]
What's happening? The first half of the template is just as important.

**Comment by FAKUgaming** [comment]
Sorry I relized after I posted it that I replaced parts of the template instead of just starting below the template.

### What happened:
basically in claimed or unclaimed lands certain people are not able to walk through doors, it is pushing them back like an invisible wall is there, even if they are the owners of the claim land. This is only happening with doors, not gates.

### What you expected:
For you to be able to enter doors in claimed land that is your own, or even through doors that were opened for you.

### Steps to reproduce:
Step 1: Place a door (in claimed land or not, with or without the support)
Step 2: Try to walk through the door. (it will prevent players from walking through it)
I personally have not been able to reproduce this on my own, except by trying to enter a door on a claimed land that does not belong to me. But it is treating the doors as if the player does not own the land. 

Side note, Is there a way to disable just that feature so players can still enter their homes using doors, instead of gates?

**Comment by bigpresh** [comment]
It sounds like there's a client/server disagreement about the state of the door - i.e. it might look open in the client, but as far as the server is concerned the door is closed because the event was cancelled but somehow the client didn't get told that.

What server version is in use?  Does the problem go away for those users if you temporarily disable NoCheatPlus?

Do you see any error messages appear in the server's log when someone affected tries to open a door and/or when they try to walk through it?

As far as I remember, GriefPrevention does not do any movement checks, so I would very strongly suspect that NoCheatPlus is blocking the movement believing that the door is closed and the client is trying to wallhack.  I never had this problem using GP and NCP together, but I've temporarily stopped using NCP when I upgraded to 1.14 as there was no updated version available and I had some problem (I can't remember what exactly) during my testing of it.

**Comment by bigpresh** [comment]
Also, to answer your last point, in two parts...

> Side note, Is there a way to disable just that feature 

Hmm.  `LockWoodenDoors: false` *should* disable checks on wooden doors, but I see you already have that in the config posted on the GH Gist.  

> so players can still enter their homes using doors, instead of gates? 

Are you saying that the players affected by this have problems with doors, but *not* with gates?

Also, OOI - does it seem to affect all variants of wooden doors, or only certain ones?

**Comment by bigpresh** [comment]
Ohhhh... in your config, I see:

`DoorsOpenDelayInSeconds: 300`

~~... that's not a Grief Prevention setting - at least, not for this (Bukkit/Spigot) version of Grief Prevention.~~ (EDIT: yes it is, I was being a muppet.)

You're not using a Sponge server and the [Sponge version of Grief Prevention](https://ore.spongepowered.org/blood/GriefPrevention), which is a totally separate project, are you?  Or some other fork?

Does this problem still happen if you temporarily remove your config.yml and let a default one be written?

**Comment by bigpresh** [comment]
> DoorsOpenDelayInSeconds: 300
>
> ... that's not a Grief Prevention setting

Hmm.  Actually, I'm wrong.  It is, or at least *was*, a setting, added (by me... ahem, I must have a great memory...) in PR #203.

I searched for it in the list of config vars defined, but didn't find it because the config var and the actual setting in the config file [have a slightly different name](https://github.com/TechFortress/GriefPrevention/blob/master/src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java#L768):

`this.config_siege_doorsOpenSeconds = config.getInt("GriefPrevention.Siege.DoorsOpenDelayInSeconds", 5*60);`

So, yes, please ignore my "that's not a GP setting" waffle - it seems I need more coffee, and/or to borrow a clue from someone.  Please do answer the other questions, though.

**Comment by FAKUgaming** [comment]
Server Version: Bukkit 1.14.4

I have not found any error messages in the server logs related to this event unfortunately.

I only assumed it would be GriefPrevention as yes it only happens with doors, stone or wooden, but gates work just fine and I saw the setting 'LockWoodenDoors' but I will test this theory as soon as I can get ahold of someone that is having the issue to test it for me, as this doesn't happen to me even if I player as a non OP character.

I will check to see if its all wooden doors, so far I have only tried with the original wooden doors and stone doors.

I do not believe I am using a sponge version as I got it off the Bukkit page

I will get back with you later today after I can get someone to assist me with testing.
